CONTRIBUTING.md

```diff
@@ -94,14 +94,15 @@ download and install anything:
 * On each documentation page, there should be an "Improve This Page" link at the very
   top.
 * Click on that link to open the respective source file (usually an `.rst` file in the
-  `doc` folder) on GitHub for editing online (you'll need a GitHub account).
+  `doc` folder or a `.py` file in the `examples` folder) on GitHub for editing online
+  (you'll need a GitHub account).
```

Comment:
Most documentation source files are in the `examples` folder, so we should mention it.